popSimManu.tex

```diff
@@ -519,68 +519,73 @@ \subsection*{Use case: comparing methods of demographic inference}
 of standardized method testing. We hope that our efforts represent the beginning of what
 will prove to be an equally longstanding and invaluable tradition in population genetics.
 
-Moreover individual research groups are often in the position of having to independently
-reimplement complex previously published models. \stdpopsim offers solutions to both of
-these issues by creating a quality controlled, easy-to-use, gateway to models and organisms
+Individual research groups are often in the position of having to independently
+reimplement complex previously published demographic models.
+Additional layers of realism (e.g., recombination maps) makes this especially daunting.
+\stdpopsim aims to lower this barrier
+by creating a quality controlled, easy-to-use, gateway to models and organisms
 that are central to modern research in genetics.
 
 We have illustrated in this paper how \stdpopsim can be used for direct
 comparisons of inferential methods on a common set of simulations. Our
-benchmarking comparisons have been limited so far, but nevertheless, they
-do reveal some broad generalizations about relative performance. For
-example, at the task of estimating $N(t)$ trajectories for single human
+benchmarking comparisons have been limited, but nevertheless
+reveal some informative comparisons. For
+example, at the task of estimating $N(t)$ trajectories for simulations of single human
 populations, we find that the sequence-based methods (\MSMC and \smcpp)
 perform somewhat better overall---at least for mid-range values of
 $t$---than the site frequency spectrum-based method (\stairwayplot)
-(Figure~\ref{fig:n_t_ragsdale} and Figure~\ref{fig:n_t_gutenkunst}), which
-tends to over-estimate oscillations in size.  By contrast, in the case of
-\emph{D.~melanogaster}, where linkage disequilibrium is considerably
-reduced, \stairwayplot outperforms the sequence-based methods
-(Figure~\ref{fig:n_t_sheehan}). In the two-population setting for human
+(Figures~\ref{fig:n_t_ragsdale} and~\ref{fig:n_t_gutenkunst}), which
+tends to over-estimate the size of oscillations.  By contrast,
+\stairwayplot outperforms the sequence-based methods
+on simulations of \textit{D.~melanogaster} populations,
+in which linkage disequilibrium is considerably reduced (Figure~\ref{fig:n_t_sheehan}).
+In simulations of two human populations
 (Figure~\ref{fig:IM_popn_human}), most methods do reasonably well at
 reconstructing the YRI history, but struggle with the more complex CEU
 history, in large part because of the restriction of constant sizes per
 population.  An exception is \smcpp, which does not have the same
 restrictions on its inferred history, and as a result does somewhat better
 with the CEU history but tends to overfit the YRI history.  The results for
-the two-population {\em Drosophila} case (Figure~\ref{fig:two_popn_fly})
+the two-population \textit{D.~melanogaster} case (Figure~\ref{fig:two_popn_fly})
 are generally similar.  
```

Comment:
technically species names are italicised not emphasized; and, we used Dmel above, so should be consistent.

Reply:
I wonder if we should just use the `stdpopsim` IDs e.g., DroMel

Reply:
only if you use HomSap above (I vote keep the organism name tho)